scipy/sparse/linalg/isolve/iterative.py

```diff
@@ -456,44 +728,28 @@ def gmres(A, b, x0=None, tol=1e-5, restart=None, maxiter=None, M=None, callback=
 
     Other parameters
     ----------------
-    x0 : {array, matrix}
+    x0: {array, matrix}
```

Comment:
the original was correct, can you revert?

Reply:
same for other formatting changes here

Reply:
Ok, I'll recover these.